src/zon.zig

```diff
@@ -250,6 +250,10 @@ pub const ZonElement = union(enum) { // MARK: Zon
 		}
 	}
 
+	pub fn append(self: *const ZonElement, value: anytype) void {
+		self.array.append(createElementFromRandomType(value, self.array.allocator.allocator));
+	}
```

Comment:
Please rebase the PR, and remove this function, since it was just added in #1190